README.md

```diff
@@ -1,10 +1,10 @@
-# Primer Interface Guidelines
+# Primer Design System documentation 
 
-Documentation for UI patterns and interaction guidelines.
+Documentation for all things Primer, including components, UI patterns, foundations, guides, and more. 
 
 ## Status
 
-This is currently a work in progress! Follow along on our [project board](https://github.com/primer/design/projects/1).
+This repository is evolving as our documentation needs change. See our [contribution guides here](https://primer.style/design/guides/contribute/documentation). 
```

Comment:
Maybe it's worth linking to this project board now that we've gotten rid of the other one?

https://github.com/orgs/github/projects/4503/views/16